draft-patton-cfrg-vdaf.md

```diff
@@ -839,37 +839,52 @@ def vec_add(left, right):
 
 ## Pseudorandom Generators {#prg}
 
+> CP TODO Update the reference implementation to match.
+
 A pseudorandom generator (PRG) is used to expand a short, (pseudo)random seed
 into a long string of pseudorandom bits. A PRG suitable for this document
 implements the interface specified in this section. Concrete constructions are
 described in {{prg-constructions}}.
 
-PRGs are defined by a class `Prg` with a single parameter:
+PRGs are defined by a class `Prg` with the follwing associated parameter:
 
 * `SEED_SIZE: Unsigned` is the size of the input seed.
 
 A concrete `Prg` implements the following class method:
 
-* `Prg.expand(seed: Bytes, info: Bytes, length: Unsigned) -> Bytes` derives a
-  byte string from the given seed and info string. The length of the output MUST
-  be `length`.
+* `Prg(seed: Bytes, info: Bytes, length: Unsigned)` constructs an instance of
+  `Prg` from the given seed and info string. The seed MUST be of length
+  `SEED_SIZE` and MUST be generated securely (i.e., it is either the output of
+  `gen_rand` or a previous invocation of the PRG).
+
+* `prg.next(length: Unsigned)` returns the next `length` bytes of output of the
+  PRG. If the seed was securely generated, the output can be treated as
+  pseudorandom.
```

Comment:
Maybe add a note about `info` and domain separation?

Reply:
Done